client/app/bundles/course/level/components/LevelRow.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl } from 'react-intl';
+import { TableHeaderColumn, TableRow, TableRowColumn } from 'material-ui/Table';
+import RaisedButton from 'material-ui/RaisedButton';
+import TextField from 'material-ui/TextField';
+import { grey300 } from 'material-ui/styles/colors';
+import styles from './../containers/LevelsLayout/LevelView.scss';
+
+class LevelRow extends React.Component {
+  static propTypes = {
+    levelNumber: PropTypes.number.isRequired,
+    experiencePointsThreshold: PropTypes.number.isRequired,
+    updateExpThreshold: PropTypes.func.isRequired,
+    sortLevels: PropTypes.func.isRequired,
+    deleteLevel: PropTypes.func.isRequired,
+  }
+
+  renderInput(levelNumber, experiencePointsThreshold) {
+    return (
+      <TextField
+        type="text"
+        name={`level_${levelNumber}`}
+        onChange={(e, newValue) => {
+          this.props.updateExpThreshold(levelNumber, newValue);
+        }}
+        onBlur={() => { this.props.sortLevels(); }}
+        value={experiencePointsThreshold}
+      />
+    );
+  }
+
+  render() {
+    const { levelNumber, experiencePointsThreshold } = this.props;
+    return (
+      <TableRow>
+        <TableHeaderColumn className={styles.deleteButtonCell}>
+          <RaisedButton
+            name={levelNumber}
+            backgroundColor={grey300}
+            icon={<i className="fa fa-trash" />}
```

Comment:
Use [Material Design icons](https://material.io/icons/) instead of FontAwesome.

[Material UI Svg Icon](http://www.material-ui.com/#/components/svg-icon) - import `material-ui/svg-icons/action/delete`